### ISSUE TITLE
Module doesn't close connection to db after module destroy

### ISSUE THREAD

**Issue by CheerlessCloud** [kpfromer/nestjs-typegoose#15]
Due to the lack of an explicit connection closure, it is necessary to add --forceExit to jest. Yes, of course it can be solved by manual get connection by token and call close method, but why not implement it as in @nest/mongoose?
https://github.com/nestjs/mongoose/blob/master/lib/mongoose-core.module.ts#L113

**Comment by kpfromer** [comment]
I think that will work to mimic @nest/mongoose implementation of disconnecting the connection. Do you have a fork that you are working on?

**Comment by duro** [comment]
@CheerlessCloud @kpfromer Any chance we could get this fork PR'd and merged? This is a big pain for testing.

**Comment by kpfromer** [comment]
@duro Yeah. I am planning on getting all the issues fixed in the next week.

**Comment by duro** [comment]
@kpfromer Great.

For anyone else, until then, a workaround is to add this to your app module:

```
export class MyModule implements OnModuleDestroy {
  constructor(private readonly moduleRef: ModuleRef) {}

  public async onModuleDestroy() {
    const connection = this.moduleRef.get<any>('DbConnectionToken', { strict: false })
    if (connection) {
      await connection.close()
    }
  }
}
```

**Comment by CheerlessCloud** [comment]
Sorry for waiting, I had a lot of work this month. I have just created PR for fix this issue #18.
